models/notes.py

```diff
@@ -9,19 +9,20 @@ class NotesModel(BaseModel):
 
     id = db.Column(db.Integer, primary_key=True)
     text = db.Column(db.Text)
-    user = db.Column(db.Integer, db.ForeignKey('users.id'))
+    userid = db.Column(db.Integer, db.ForeignKey('users.id'),
+        nullable=False)
 
     ticketid = db.Column(db.Integer, db.ForeignKey('tickets.id'),
         nullable=False)
 
     def json(self):
-        user = UserModel.find_by_id(self.user)
+        userobj = models.user.UserModel.find_by_id(self.userid)
 
         return {
             'id':self.id,
             'ticketid': self.ticketid,
             'text': self.text,
-            'user': user.full_name(),
+            'user': userobj.full_name(),
```

Comment:
Now that the relationship is setup., We should be able to do.
```suggestion
            'user': self.user.full_name(),
```

Reply:
I didn't realize the relationship made that possible.